fixit/cli/run_rules.py

```diff
@@ -87,33 +100,48 @@ def get_formatted_reports_for_path(
     return [opts.formatter.format(rr) for rr in raw_reports]
 
 
-def main(raw_args: Sequence[str]) -> int:
-    parser = argparse.ArgumentParser(
-        description=(
-            "Validates your lint rules by running them against the specified, "
-            + "directory or file(s). This is not a substitute for unit tests, "
-            + "but it can provide additional confidence in your lint rules.\n"
-            + "If no lint rules or packages are specified, runs all lint rules "
-            + "found in the packages specified in `fixit.config.yaml`."
-        ),
-        parents=[
-            get_paths_parser(),
-            get_rules_parser(),
-            get_use_ignore_comments_parser(),
-            get_skip_ignore_byte_marker_parser(),
-            get_compact_parser(),
-            get_multiprocessing_parser(),
-        ],
-    )
-
+def _parser_arguments(
+    parser: Union[argparse._SubParsersAction, argparse.ArgumentParser],
+    sub_parser: bool = True,
```

Comment:
`sub_parser` is not needed.
You can simply do to help Pyre understand the type and it's more readable.
```
if isinstance(parser, argparse._SubParsersAction):
    ...
else:
    ...
```

Reply:
I tried this 
```
if isinstance(parser, argparse._SubParsersAction):
    ...
else:
    ...
```
but parser is always of type `argparse.ArgumentParser `, that's why I use `sub_parser`